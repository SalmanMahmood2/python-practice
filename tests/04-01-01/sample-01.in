Apple
Banana
Pear

Orange
Banana
Pear
Grapes
