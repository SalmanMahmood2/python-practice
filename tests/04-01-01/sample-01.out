Not in stock!
In stock!
In stock!
Not in stock!